Formula/geogram.rb

```diff
@@ -0,0 +1,25 @@
+class Geogram < Formula
+  desc "Programming library of geometric algorithms"
+  homepage "http://alice.loria.fr/software/geogram/doc/html/index.html"
+  url "https://gforge.inria.fr/frs/download.php/file/37635/geogram_1.6.6.tar.gz"
+  sha256 "08211b1d6f21e14701e3fd5c79adbe331cdf66b8af84efdb54cd7048244691b5"
+  depends_on "cmake" => :build
```

Comment:
Should be an empty line between `sha256` and the first `depends_on`.

Reply:
Thanks!